app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -801,6 +802,31 @@ class BrowserTabFragment :
         decorator.incrementTabs()
     }
 
+    private fun openAppLinkDialog(
+        appIntent: Intent?,
+        excludedComponents: List<ComponentName>?,
+        url: String? = null,
+        headers: Map<String, String> = emptyMap()
+    ) {
+        context?.let {
```

Comment:
are you checking in the fragment is attached or if the fragment hasn't been destroyed?